CMakeLists.txt

```diff
@@ -147,12 +103,33 @@ install(
   COMPONENT freeciv21)
 
 if(WIN32 OR MSYS OR MINGW)
-  # Install the libraries compiled in the section above
-  install(
-    PROGRAMS
-    ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS}
-    DESTINATION ${CMAKE_INSTALL_BINDIR}
-    COMPONENT freeciv21)
+  install(CODE [[
+    message(STATUS "Collecting dependencies for freeciv21 executables...")
+
+    # Take a variable that is available at "install" time and repurpose
+    string(REGEX REPLACE "objdump.exe" "" MINGW_PATH ${CMAKE_OBJDUMP})
+
+    # Function to analyze the third party dll files linked to the exe's
+    #   Uses the repurposed variable from above to tell the function where
+    #   the dll files are located. Ignores dll's that come with Windows.
+    file(GET_RUNTIME_DEPENDENCIES
```

Comment:
One last comment: this command is new in 3.16 so we need to require it on line 1 (but please keep 3.12 support for Linux, as it's what the Longturn server has).

Reply:
Handled in second commit